### ISSUE TITLE
TestGolden fails w/ Go 1.14 beta1

### ISSUE THREAD

**Issue by eclipseo** [golang/protobuf#1029]
**What version of protobuf and what language are you using?**
1.3.3, go 1.14 beta 1, Fedora Rawhide

**What did you do?**
Run tests

**What did you expect to see?**
No failure.

**What did you see instead?**
```
Testing    in: /builddir/build/BUILD/protobuf-1.3.3/_build/src
         PATH: /builddir/build/BUILD/protobuf-1.3.3/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/protobuf-1.3.3/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/golang/protobuf/version=1.3.3 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/golang/protobuf
github.com/golang/protobuf/jsonpb
PASS
ok  	github.com/golang/protobuf/jsonpb	0.016s
github.com/golang/protobuf/proto
--- Dump of b ---
  0: t=  1 bytes [7] 08 01 12 .. 52 6f 62
  9: t=  1 bytes [7] 08 04 12 .. 49 61 6e
 18: t=  1 bytes [8] 08 08 12 .. 61 76 65
2020/02/04 18:33:47 proto: don't know how to compare extension 200 of test_proto.MyMessage
PASS
ok  	github.com/golang/protobuf/proto	0.054s
github.com/golang/protobuf/protoc-gen-go
--- FAIL: TestGolden (0.39s)
    golden_test.go:346: RUNNING:  protoc --plugin=protoc-gen-go=/tmp/go-build385470712/b001/protoc-gen-go.test -Itestdata --go_out=plugins=grpc,paths=source_relative:/tmp/proto-test639569157 testdata/my_test/test.proto
    golden_test.go:349: my_test/test.proto:40:1: warning: Import multi/multi1.proto is unused.
        
    golden_test.go:112: golden file differs: issue780_oneof_conflict/test.pb.go
        --- testdata/issue780_oneof_conflict/test.pb.go	2020-01-28 18:10:22.000000000 +0000
        +++ /tmp/proto-test639569157/issue780_oneof_conflict/test.pb.go	2020-02-04 18:33:49.848538575 +0000
        @@ -89,7 +89,9 @@
         	proto.RegisterType((*Foo)(nil), "oneoftest.Foo")
         }
         
        -func init() { proto.RegisterFile("issue780_oneof_conflict/test.proto", fileDescriptor_48462cafc802a68e) }
        +func init() {
        +	proto.RegisterFile("issue780_oneof_conflict/test.proto", fileDescriptor_48462cafc802a68e)
        +}
         
         var fileDescriptor_48462cafc802a68e = []byte{
         	// 107 bytes of a gzipped FileDescriptorProto
FAIL
exit status 1
FAIL	github.com/golang/protobuf/protoc-gen-go	0.636s
```

**Comment by dsnet** [comment]
Thanks for the report, but these tests will be gone soon. In fact they have already been removed in the `api-v1` branch.

**Comment by neild** [comment]
I think we should fix this. It's annoying when we're so close to replacing the test, but it's also annoying for users to deal with spuriously broken tests.

**Comment by neild** [comment]
Fixed in v1.3.4.
